core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -226,6 +228,7 @@ oppia.directive('audioTranslationBar', [
               saveContentIdsToAudioTranslationChanges();
               AlertsService.addSuccessMessage(
                 'Succesfuly uploaded recorded audio.');
+              $scope.checkSavingProcess = false;
```

Comment:
What if the audio didn't save successfully? Take care of that case too.

Reply:
I am not getting where this variable is declared "recorder.status.isConverting"  in order to change anything to change inside 
                     <ng-audio-recorder id="<[RECORDER_ID]>"
                     on-record-start="onRecordStart()"
                     on-conversion-complete="onConversionComplete()"
                     audio-model="audioBlob"
                     time-limit="recordingTimeLimit">
Need help

Reply:
Sorry, I didn't understand your question clearly around this line 
> in order to change anything to change inside

 Still regarding `recorder.status.isConverting` see [here](https://github.com/sathyapulse/angular-recorder#properties-1).

Reply:
solved this